k8s/demo/common-overlay/ia/kustomization.yaml

```diff
@@ -11,7 +11,8 @@ patchesStrategicMerge:
 - ../../../namespaces/ia/ia-case-api/demo.yaml
 - ../../../namespaces/ia/ia-bail-case-api/demo.yaml
 - ../../../namespaces/ia/ia-case-notifications-api/demo.yaml
-- ../../../namespaces/ia/ia-case-documents-api/demo.yaml
```

Comment:
Why removal of` ia-case-documents-api` ?

Reply:
discussed with Rajesh, duplicate entry

Reply:
its a duplicate entry